interchain/scripts/state.json

```
@@ -290,5 +314,31 @@
         "abstract:version-control": "terra1d87wpuvjgqe8a5qytneyd8puc7y2uan6tcr6mwq873trjm8v2pzs7t593t"
       }
     }
+  },
+  "xion": {
+    "xion-testnet-1": {
+      "code_ids": {
+        "abstract:account-factory": 515,
+        "abstract:ans-host": 513,
+        "abstract:ibc-client": 519,
+        "abstract:ibc-host": 520,
+        "abstract:manager": 517,
+        "abstract:module-factory": 516,
+        "abstract:proxy": 518,
+        "abstract:version-control": 514
+      },
```

```suggestion
      "code_ids": {

        "abstract:account-factory": 654,
        "abstract:ans-host": 657,
        "abstract:ibc-client": 660,
        "abstract:ibc-host": 661,
        "abstract:manager": 658,
        "abstract:module-factory": 655,
        "abstract:proxy": 659,
        "abstract:version-control": 656
      },
```
from `release-0.22` branch migrated xion